[IMP] ribbon color customization

jabibi opened the pull request:
## web_environment_ribbon

The ribbon color and background-color can be modified through system parameters:
- ribbon.color
- ribbon.background.color

pedrobaeza commented on README.md:
```diff
@@ -31,7 +31,7 @@ addon | version | summary
 [web_dom_model_classes](web_dom_model_classes/) | 8.0.1.0.0 | Allows small UI changes with simple CSS
 [web_easy_switch_company](web_easy_switch_company/) | 8.0.1.0.0 | Multicompany - Easy Switch Company
-[web_environment_ribbon](web_environment_ribbon/) | 8.0.0.1.0 | Web Environment Ribbon
+[web_environment_ribbon](web_environment_ribbon/) | 8.0.1.0.0 | Web Environment Ribbon
```
Don't change this. It's automatically changed when this module gets merged.

pedrobaeza replied:
You have to clean your commits to remove the diff in this file, or there will be conflicts for sure.

jabibi replied:
Ok, I didn't know

pedrobaeza commented:
Write here when you finish to review full PR. Thanks for the contribution!

jabibi commented:
Just done it
Hope this will be usefull

jabibi committed:
[IMP] web_environment_ribbon: add color customization
The ribbon color and background-color can be modified through system
parameters:
 * ribbon.color
 * ribbon.background.color
 
Added Copyright in .js
Delete duplicate entries in css

jabibi committed:
[UPD] web_environment_ribbon: README.rst update and version fix
* Fix module version (OCA Guidelines)
* Update README file with latest modifications

andhit-r commented on web_environment_ribbon/README.rst:
```diff
@@ -12,46 +12,41 @@ Mark a Test Environment with a red ribbon on the top left corner in every page
+  Parameters) To hide the ribbon, set this parameter to "False" or delete it.
+* You can customize the ribbon color and background color through system
+  parameters: "ribbon.color", "ribbon.background.color". Fill with valid CSS
+  colors or just set to "False" to use default values.
```
can we use hex notation for ribbon.background.color?

jabibi replied:
Yes, all CSS values for colors are accepted

andhit-r commented:
Tested on runbot :+1:

pedrobaeza commented:
Tested also in runbot and working :+1:

lfreeke commented:
Tested on runbot :+1:
